src/mprisbluezproxy.h

```diff
@@ -0,0 +1,132 @@
+// -*- c++ -*-
+
+/*!
+ *
+ * Copyright (C) 2018 Jolla Ltd.
+ *
+ * Contact: Jarkko Lehtoranta <jarkko.lehtoranta@jolla.com>
+ * Author: Jarkko Lehtoranta <jarkko.lehtoranta@jolla.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+
+#ifndef MPRISBLUEZPROXY_H
+#define MPRISBLUEZPROXY_H
+
+#include <MprisQt>
+#include <Mpris>
+#include <MprisController>
+#include <MprisManager>
+#include <MprisPlayer>
+
+#include <QObject>
+#include <QDBusConnection>
+#include <QDBusObjectPath>
+#include <QDBusServiceWatcher>
+
+#include <QtCore/QByteArray>
+#include <QtCore/QList>
+#include <QtCore/QMap>
+#include <QtCore/QObject>
+#include <QtCore/QSharedPointer>
+#include <QtCore/QString>
+#include <QtCore/QStringList>
+#include <QtCore/QVariant>
+
+typedef QMap<QString, QDBusVariant> InterfaceList;
+typedef QMap<QString, QVariantMap> AdapterList;
+typedef QMap<QDBusObjectPath, AdapterList> ManagedObjectList;
+
+Q_DECLARE_METATYPE(InterfaceList)
+Q_DECLARE_METATYPE(AdapterList)
+Q_DECLARE_METATYPE(ManagedObjectList)
+
+class MprisPlayer;
+class MPRIS_QT_EXPORT MprisBluezProxy : public QObject
+{
+    Q_OBJECT
+    Q_PROPERTY(bool useSystemBus READ useSystemBus)
+
+public:
+    MprisBluezProxy(QObject* parent = 0);
+    ~MprisBluezProxy();
+
+public slots:
+    bool useSystemBus() const {
+        return true;
+    }
```

Comment:
In addition to above, this is now both property and slot. Doubt it needs either.